### DISCUSSION TITLE
App on iPadOS

### DISCUSSION THREAD

**Discussion by sercontri** [coteditor/CotEditor#1502] [Ideas]
After having enjoyed the app in amateur mode on macOS, I had to retire the Mac and I went to iPad Pro. I don't know if there is a future idea of being able to migrate this app to the iPad even if it is in a basic way and it can be paid to support the creator. I also don't know if there is anything already available through TestFlight...

I open a debate in case there are more people with the same concern as me. Thank you for the work done. For me it's 10.

**Comment by 1024jp** [comment]
Although I understand the demand, it is still quite hard to port CotEditor to iOS/iPadOS since it fully relies on AppKit framework. In addition, CotEditor is a project maintained by only one person; therefore, it required significant time for such a large change. I feel CotEditor is not ready for it.

**Comment by MiKhelangelo** [comment]
While iOS does not provide a direct equivalent to macOS's AppleScript, you can still achieve similar functionality by leveraging iOS-specific capabilities such as URL schemes, server-side APIs, and app extensions. Won’t be easy but I feel like it’s achievable. Best editor I’ve ever came across ever.

**Comment by sercontri** [comment]
Hello, I’m reopening the discussion... I still work with a Mac and an iPad, and I really miss it on my iPad. A few years have passed, and I don’t know how things are now regarding being able to enjoy coteditor on iPad. Best regards, and many thanks for the app!

**Reply by 1024jp** [reply]
Unfortunately, the situation has not been changed.
